components/renku_data_services/users/api.spec.yaml

```
@@ -356,6 +369,12 @@ components:
       type: string
       minLength: 1
       maxLength: 5000
+    SecretKind:
+      description: Kind of secret
+      type: string
+      enum:
+        - "general"
+        - "storage"
```

In my experience, enums are usually in all caps:

```suggestion
        - "GENERAL"
        - "STORAGE"
```

---

Clarified -- in this repo lowercase is used for enums.